modules/darwinosl/darwinosl-grammar.ym

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) 2023 Hofi <hofione@gmail.com>
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 as published
+ * by the Free Software Foundation, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * As an additional exemption you are allowed to compile & link against the
+ * OpenSSL libraries as published by the OpenSSL project. See the file
+ * COPYING for details.
+ *
+ */
+
+%code top {
+#include "darwinosl-parser.h"
+}
+
+%code {
+
+#include "darwinosl-source.h"
+#include "cfg-parser.h"
+#include "syslog-names.h"
+#include "messages.h"
+#include "plugin.h"
+#include "cfg-grammar-internal.h"
+
+#include <string.h>
+
+}
+
+%define api.prefix {darwinosl_}
+
+/* this parameter is needed in order to instruct bison to use a complete
+ * argument list for yylex/yyerror */
+
+%lex-param {CfgLexer *lexer}
+%parse-param {CfgLexer *lexer}
+%parse-param {LogDriver **instance}
+%parse-param {gpointer arg}
+
+/* INCLUDE_DECLS */
+
+%token KW_DARWIN_OSL
+%token KW_FILTER_PREDICATE
+%token KW_GO_REVERSE
+%token KW_DO_NOT_USE_BOOKMARK
+%token KW_MAX_BOOKMARK_DISTANCE
+%token KW_LOG_FETCH_DELAY
+%token KW_LOG_FETCH_RETRY_DELAY
+
```

Comment:
I would lose the "log-" prefix from these keywords, I know that log-iw-size(), log-fifo-size() at all use it, but I think low-level, driver specific options should not.

Reply:
log-iw-size, log-fifo-size is the smaller issue, my problem here is that we already have `log-fetch-limit` that is the reason i used the `log-fetch`  (not simply the `log-` prefix)